addons/crm/models/crm_lead.py

```diff
@@ -2121,7 +2121,13 @@ def message_new(self, msg_dict, custom_values=None):
             defaults['priority'] = msg_dict.get('priority')
```

Comment:
Would be nice to test mail auto-assignment. You can mock receiving an email with `format_and_process` from `MockEmail`. But feel free to just call `message_new` manually with the right config, I'd say that's enough of a test for our purposes.